frontends/main/src/app-pages/DashboardPage/CoursewareDisplay/OrganizationCards.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react"
+import Image from "next/image"
+import graduateLogo from "@/public/images/dashboard/graduate.png"
+import { Stack, Typography, styled, theme } from "ol-components"
+import { useQuery } from "@tanstack/react-query"
+import { DashboardCardRoot } from "./DashboardCard"
+import { mitxUserQueries } from "api/mitxonline-hooks/user"
+import { contractQueries } from "api/mitxonline-hooks/contracts"
+import { ButtonLink } from "@mitodl/smoot-design"
+import { organizationView } from "@/common/urls"
+
+const Wrapper = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  gap: "40px",
+  marginTop: "32px",
+  padding: "24px 32px 32px 32px",
+  backgroundColor: theme.custom.colors.white,
+  borderBottom: `1px solid ${theme.custom.colors.red}`,
+  boxShadow: "0px 4px 8px 0px rgba(19, 20, 21, 0.08)",
+  borderRadius: "8px",
+  [theme.breakpoints.down("md")]: {
+    border: `1px solid ${theme.custom.colors.lightGray2}`,
+    backgroundColor: "rgba(243, 244, 248, 0.60);", // TODO: use theme color
+    marginTop: "16px",
+    padding: "0",
+    gap: "8px",
+  },
+}))
+
+const CardRootStyled = styled(DashboardCardRoot)({
+  borderRadius: "8px",
+  boxShadow: "0px 1px 6px 0px rgba(3, 21, 45, 0.05)",
+})
```

Comment:
well i was going to suggest just copying the styles from DashboardCard, but i guess it's reused in Figma, so presumably they'll change together.